index.markdown

```diff
@@ -933,22 +991,42 @@ It's worth a trip after your first paycheque.
   You can easily spend a whole day here.
 * [Planet Granite](http://planetgranite.com/locations/sunnyvale/sv_faq.php) for
   those who want to indoor rock climb
+* [Go Karting](https://www.k1speed.com/santa-clara-location.html)
 * Stanford football game - Eric says they're just like the movies!
 
 #### Trips
 
 * Yosemite National Park
-  * Upper Yosemite falls is a long hike (4 hours) but its well worth the views.
+  * If you want to stay in a cabin, make sure you book several weeks in advance since
+    they fill up really fast.
+  * Booking hotels in nearby towns that are about an hour away, like Mariposa, is also
+    an option, but they fill up fast as well.
+  * If you want a breathtaking view with minimal walking, check out Glacier Point.
+  * Upper Yosemite Falls is a long hike (4 hours) but its also well worth the views.
     Fairly popular route. Bring water and good shoes and be prepared to be tired.
   * If you're willing to spend two days hiking, you can hike the famous half
     dome.
+
+* Highway One
+  * Very scenic highway that runs along California's coastline.
+
+* Mono Lake
+  * If you end up in this area, Ankit highly recommends hiking the Parker Lake trail,
+    a 4-mile hike with amazing views of the Sierras.
+  * There's also a historic Wild West [ghost town](https://en.wikipedia.org/wiki/Bodie,_California)
+    in the area.
+
+* Los Angeles
+  * Lots of things to see such the famous Hollywood sign, and beaches such as Venice
+    Beach, where all the nightlife is.
+
 * Las Vegas
   * Cirque de Soleil shows are a must.
   * Grand Canyon if you have a day or two. Drive out is about 4 hours each way
     however so be warned.
   * Hoover Dam has a nice indoor tour where you get to see the generators and
     water tunnels. Also a beautiful sight.
-  * Casinos and restaurants of course on the strip.
+  * Casinos and restaurants of course on the strip, for those 21 and over.
```

Comment:
I'll remove it altogether, I think the 21 or older part is pretty self explanatory.